hbase-server/src/main/java/org/apache/hadoop/hbase/master/MobFileCleanerChore.java

```diff
@@ -39,27 +39,33 @@
  * mob files.
  */
 @InterfaceAudience.Private
-public class ExpiredMobFileCleanerChore extends ScheduledChore {
+public class MobFileCleanerChore extends ScheduledChore {
```

Comment:
I don't see any tests for

a) properly removing obsolete files as expected
b) not removing mob files from prior to the reference accounting added in this change.

Reply:
Created:
https://issues.apache.org/jira/browse/HBASE-23188